edb/server/protocol/auth_ext/http.py

```diff
@@ -1240,7 +1244,7 @@ def _get_data_from_reset_token(self, token: str) -> Tuple[str, str, str]:
 
     def _get_data_from_verification_token(
         self, token: str
-    ) -> Tuple[str, float, Optional[str], Optional[str]]:
+    ) -> Tuple[str, float, str, Optional[str], Optional[str]]:
```

Comment:
This is borderline ready to be a dataclass, I think, but this is still fine for now. 😆

Reply:
As far as I can tell, tuples can contain up to 2^63 elements, so we should be able to pack in a few more values yet. 😄